sys-apps/shadow/shadow-4.12.3.ebuild

```diff
@@ -242,3 +245,28 @@ pkg_preinst() {
 	rm -f "${EROOT}"/etc/pam.d/system-auth.new \
 		"${EROOT}/etc/login.defs.new"
 }
+
+pkg_postinst() {
```

Comment:
Just curious.
Is that intended that `pkg_postinst` is not being dropped any more?